test/models/report_test.rb

```diff
@@ -1,7 +1,26 @@
 require 'test_helper'
 
 class ReportTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  test "report saved" do
+    original_report_total = Report.count
+    r = Report.new(content: "New report")
+    r.save
+    assert_equal (original_report_total + 1), Report.count
+  end
+
+  test "accepted report passes accepted? method" do
+    r = reports(:first_report)
+    assert r.accepted_by_project_owner == nil
```

Comment:
you can improve this test by asserting that the report is not accepted, instead of testing if accepted_by_project_owner is nil. 

    r = report(:first_report)
    assert_not r.accepted?
    r.accepted_by_project_owner = true
    asset r.accepted?